traffic_portal/test/integration/PageObjects/LoginPage.po.ts

```diff
@@ -29,24 +29,38 @@ export class LoginPage extends BasePage{
     private config = require('../config');
     private randomize = this.config.randomize;
 
-    async Login(userName: string, password: string ){
-        if(userName == 'admin'){
-            await this.txtUserName.sendKeys(userName)
-            await this.txtPassword.sendKeys(password)
+    async Login(login){
```

Comment:
Missing typing on `Login` function. You could just do `login: {password: string, username: string}`, or make a separate type (which is just what I like to do, personally) and do

```typescript
interface LoginData {
    password: string;
    username: string;
}

//...

async function Login(login: LoginData): Promise<boolean> {
    //...
}
```

Reply:
Used interface like you !!!